README.md

```diff
@@ -91,12 +91,49 @@ gem "tailwindcss-ruby", "~> 3.4"
 
 First, update to `tailwindcss-rails` v4.0.0 or higher. This will also ensure you're transitively depending on `tailwindcss-ruby` v4.
 
-``` html
+```html
 # Gemfile
 gem "tailwindcss-rails", "~> 4.0" # which transitively pins tailwindcss-ruby to v4
 ```
+To replace the class names accordingly for v4, go to ``config/tailwind.config.js`` update the ``content`` part to:
+
+```js
+  content: [
+    '../public/*.html',
+    '../app/helpers/**/*.rb',
+    '../app/javascript/**/*.js',
+    '../app/views/**/*.{erb,haml,html,slim}'
+  ],
+```
+(Just add an additional ``.`` to all the paths referenced)
```

Comment:
I mentioned this in the Discussions thread, but I think this is a bug that should be reported upstream. If you have a clean repro, would you please report that the path is being interpreted as relative to the config file in the upgrade tool (and not CWD as the CLI correctly does)?